src/EnergyPlus/main.cc

```diff
@@ -63,6 +63,8 @@ using EnergyPlus::CommandLineInterface::ProcessArgs;
 int
 main( int argc, const char * argv[] )
 {
+	// the following line is only needed when debugging issues related to NaN in Visual Studio. See https://github.com/NREL/EnergyPlus/wiki/Debugging-Tips
+	// unsigned int fp_control_state = _controlfp( _EM_INEXACT, _MCW_EM );
```

Comment:
No we don't want it all the time since the debugger looking for occurrences of Nan's is significantly slower. 
I am not sure it is worth this much effort, maybe I should just delete the commented out lines and let people find the wiki entry on the topic. Would that be better?